src/cls/IPM/ResourceProcessor/FileCopy.cls

```diff
@@ -39,7 +39,12 @@ Method OnBeforePhase(pPhase As %String, ByRef pParams) As %Status
 		}
 
 		If (pPhase = "Activate") && (..InstallDirectory '= "") && ('..Defer) {
```

Comment:
@isc-jlechtne, it seems like everything in this if statement and the one below is identical. Perhaps that can be separated out into a helper method that cleans up the source directory, sets the target, and runs DoCopy(...) iff the phase is Activate and the Installer directory is not empty. 

Or at least, break out the logic to clean up the source directory?